release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -858,6 +858,16 @@ clouds:
 
 To use application credentials with your cluster as a {rh-openstack} administrator, create the credentials. Then, use them in a `clouds.yaml` file when you install a cluster. Alternatively, you can create the `clouds.yaml` file and rotate it into an existing cluster.
 
+[id="ocp-4-12-hcp"]
+=== Hosted control planes (Technology Preview)
+
+[id="ocp-4-12-hcp-versioning"]
+==== Versioning for hosted control planes
+
+With each major, minor, or patch version release of {product-title}, 2 components of hosted control planes are released: the HyperShift Operator and the command-line interface (CLI). Any HyperShift Operator that is released for a minor _n_ version of {product-title} must work with _n_, _n-1_, and _n-2_ versions of {product-title}. However, because the hosted control planes feature was introduced as a Technology Preview in {product-title} version 4.11, it is currently available with versions only as far back as 4.11.
```

Comment:
I haven't seen the `_n_` formating used for OCP versions. I have seen a few instances where we just use `z-stream`. Might need more of a discussion. Did you use this format before/was it suggested?

Reply:
That format was suggested by engineering. I agree it isn't the clearest, as _n_ is supposed to represent a minor version number, like 4.12. It's not referring to a z-stream. 

I wanted to add an example with real version numbers, but the PM and I decided against it because hosted control planes were introduced in 4.11, so it would be misleading if I wrote, "For example, a HyperShift Operator that is released for version 4.12 must work with version 4.12, 4.11, and 4.10." (Because hosted control planes weren't available in 4.10, that statement isn't true.) 

I have a to-do for myself to update this topic for 4.13 when I can honestly write, "For example, a HyperShift Operator that is released for version 4.13 must work with version 4.13, 4.12, and 4.11."

I hope that makes sense!

Reply:
I updated the draft to clarify the _n_ wording. Thank you, @kelbrown20!